napari/utils/config.py

```diff
@@ -19,47 +17,13 @@ def _set(env_var: str) -> bool:
 """
 Experimental Features
 
-Async Loading
--------------
-Image layers will use the ChunkLoader to load data instead of loading
-the data directly. Image layers will not call np.asarray() in the GUI
-thread. The ChunkLoader will call np.asarray() in a worker thread. That
-means any IO or computation done as part of the load will not block the
-GUI thread.
-
-Set NAPARI_ASYNC=1 to turn on async loading with default settings.
-
-Octree Rendering
-----------------
-Image layers use an octree for rendering. The octree organizes the image
-into chunks/tiles. Only a subset of those chunks/tiles are loaded and
-drawn at a time. Octree rendering is a work in progress.
-
-Enabled one of two ways:
-
-1) Set NAPARI_OCTREE=1 to enabled octree rendering with defaults.
-
-2) Set NAPARI_OCTREE=/tmp/config.json use a config file.
-
-See napari/utils/_octree.py for the config file format.
-
 Shared Memory Server
 --------------------
 Experimental shared memory service. Only enabled if NAPARI_MON is set to
 the path of a config file. See this PR for more info:
 https://github.com/napari/napari/pull/1909.
 """
 
-# Config for async/octree. If octree_config['octree']['enabled'] is False
-# only async is enabled, not the octree.
-octree_config = get_octree_config()
```

Comment:
I didn't catch it before, but these are actually part of the napari public API (unless `utils` counts as some special exception I'm unaware of?) and so should be deprecated before being removed.

Reply:
FYI, you can deprecate a module attribute by using the module level `__getattr__` as described in [PEP 562](https://peps.python.org/pep-0562/).

Reply:
That's difficult because we are reusing the old async environment variable. I might be able to work it out since the variable `async_loading` (defined a few lines below) is used everywhere else.